lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -2033,6 +2032,10 @@ Value FIRRTLLowering::getPossiblyInoutLoweredValue(Value value) {
 /// This returns a null value for FIRRTL values that cannot be lowered, e.g.
 /// unknown width integers.
 Value FIRRTLLowering::getLoweredValue(Value value) {
+  // Allow for optional parameters
+  if (!value)
+    return value;
```

Comment:
Inserting an additional null check just for optional enable seems over kill. Can we do this in the call site `op.getEnable()? getLoweredValue(op.getEnable()): Value()`?